frontend/src/store/eceweApp.js

```diff
@@ -209,12 +215,20 @@ export const useEceweAppStore = defineStore('eceweApp', {
       return index >= 0 ? this.facilities[index].eceweApplicationId : null;
     },
     getOptInOrOut(facilityId) {
-      if (this.eceweModel.fundingModel == this.fundingModelTypes[0].id) {
+      const appStore = useAppStore(); // Access the other store
+      if (
+        appStore.getLanguageYearLabel !== PROGRAM_YEAR_LANGUAGE_TYPES.FY2025_26 &&
+        this.eceweModel.fundingModel == this.fundingModelTypes[0].id
+      ) {
         return 0;
       } else {
         const index = this.facilities.map((facilty) => facilty.facilityId).indexOf(facilityId);
```

Comment:
should we just use ?.find instead? e.g.: const facility = this.facilities?.find(fac => fac.facilityId === facilityId). return facility ? facility.optInOrOut : null

Reply:
done :)